app/javascript/react/components/RestaurantTile.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Link } from "react-router-dom"
+
+const RestaurantTile = (props) => {
+
+  return (
+      <h1>
+        <Link to={`restaurant/${props.id}`}>
+          {props.name}
+        </Link > 
+      </h1>
+  )
+}
```

Comment:
Ill allow this through, but two quick things:
- ideally, h# tags shouldnt have children. in this case, it will have a <a> child
- ideally, there should only be one h1 on the page. instead of using h1 for the size of the text, you should prefer CSS to style the font weight and size of text

again, Ill allow this but I encourage an update for best practices